### ISSUE TITLE
[web-extension:manifest] Entrypoint output for src/popup.html (src/popup.html) not found

### ISSUE THREAD

**Issue by mklueh** [aklinker1/vite-plugin-web-extension#78]
Hello,

I'm getting this error out of the sudden

> [web-extension:manifest] Entrypoint output for src/popup.html (src/popup.html) not found

and the browser does not start anymore.

does anyone have an idea what is going wrong?

```
{
  "{{chrome}}.manifest_version": 3,
  "{{firefox}}.manifest_version": 2,
  "icons": {
    "16": "icon/16.png",
    "32": "icon/32.png",
    "48": "icon/48.png",
    "96": "icon/96.png",
    "128": "icon/128.png"
  },
  "{{chrome}}.action": {
    "default_popup": "src/popup.html"
  },
  "{{firefox}}.browser_action": {
    "default_popup": "src/popup.html"
  },
  "background": {
    "service_worker": "src/background.ts"
  },
  "options_page": "src/options.html",
  "permissions": [
    "alarms",
    "storage",
    "identity",
    "activeTab",
    "contextMenus",
    "videoCapture",
    "desktopCapture",
    "notifications",
    "syncFileSystem",
    "{{firefox}}.<all_urls>"
  ]
}
```

![image](https://user-images.githubusercontent.com/1553757/223935517-fe19f16f-aa36-49a2-a9b7-d06ff79c9bd6.png)

![image](https://user-images.githubusercontent.com/1553757/223935832-7d870edd-c0cd-4ddb-a159-1fa0099ae717.png)

**Comment by aklinker1** [comment]
@mklueh Could you run this command and share the output?

```sh
npx envinfo --system --browsers --binaries --npmPackages vite,vite-plugin-web-extension
```

**Comment by mklueh** [comment]
Hi @aklinker1 

this is the output

```

  System:
    OS: Linux 5.15 Ubuntu 20.04.5 LTS (Focal Fossa)
    CPU: (5) x64 Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
    Memory: 2.38 GB / 9.72 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.19.1 - /usr/bin/node
    Yarn: 3.4.1 - ~/.yarn/bin/yarn
    npm: 8.19.4 - ~/workspace/my-project/node_modules/.bin/npm
  Browsers:
    Chrome: 110.0.5481.177
  npmPackages:
    vite: ^4.1.4 => 4.1.4
    vite-plugin-web-extension: ^2.0.2 => 2.0.2
```

**Comment by aklinker1** [comment]
Hmm, darn not windows. I just fixed a problem like this for windows, and you're also on the latest version...

All your screenshots look correct. Without looking at your full configuration, I can't tell what's causing this error. Can you share a git repo with a minimal reproduction?

**Comment by mklueh** [comment]
@aklinker1 I'll do that as soon as I have time.

But to be clear, it worked one week ago 😃 not sure if it was an older version back then. I might try to downgrade and try again

**Comment by aklinker1** [comment]
I was able to reproduce this. It's because you're using Vite 4, v2 only supports vite 3. Run `npm i vite@3.2.5` and it will work for you. I'm looking into vite 4 support now. #67

**Comment by aklinker1** [comment]
This has been fixed in `3.0.0` by adding vite 4 support. Just update the plugin version, and you should be good to go.

```sh
pnpm i vite-plugin-web-extension@latest
```

**Comment by mklueh** [comment]
@aklinker1 nice, thank you very much :)
